### PR TITLE
All new Data Flow Analysis for @live

### PR DESCRIPTION
**Pull request by WalterBright**
The new one is simpler and does non-lexical lifetimes much better.

### CODE REVIEW

File: changelog/ob.md [hunk 1]
```diff
@@ -100,34 +102,18 @@ until after the last use of the borrowed pointer.
...
-A Lent pointer loaned out its value to a Borrowed pointer. While
-in the Lent state the pointer cannot have its value read or written
-to.
-A Lent pointer must not be `scope` and must be a pointer to mutable.
-
-5. Readonly
+4. Readonly
 
 A Readonly pointer acquires its value from an Owner. Afterwards,
 the Owner enters the View state.
```
Comment (rikkimax) [inline changelog/ob.md]: What is a 'View' pointer ? :P

Comment (WalterBright) [reply changelog/ob.md]: A vestige of the previous version. Removed.

**Comment by WalterBright** [comment]
The buildkite error is unrelated.

**Comment by adamdruppe** [comment]
random barely related question but what does `@live` mean? is it pronounced like "alive" or like "he'll live"?

i know what the idea is behind it but the name is kinda weird especially since i have ever heard it spoken out loud.

**Comment by WalterBright** [comment]
> what does @live mean?

It tracks the "liveness" of pointers.

> is it pronounced like "alive" or like "he'll live"?

As in "live range". "alive"

**Pull request merged**